package.json

```diff
@@ -51,7 +51,7 @@
     "test:smoke": "node ./bin/mocha --no-config test/smoke/smoke.spec.js"
   },
   "dependencies": {
-    "ansi-colors": "^4.1.3",
+    "ansi-colors": "^4.1.1",
```

Comment:
Why keep it the same? I think we should bump it?

Reply:
Oh, oops, will revert any actual version drops.

Reply:
Discussed with @Uzlopak in a regular meeting: yes, we want to make sure the version we test with (listed in our `package-lock.json`) is the minimum version users receive. 

I was initially hesitant about this because I didn't want to force users to upgrade in case that's a blocker for some and/or causes duplicate package versions. But the making sure we're not using a newer version that lacks some of the features of a user's older version convinced me. 😄